all-requirements.txt

```diff
@@ -10,3 +10,6 @@ tornado==4.2
 
 # For simpl.middleware.cors
 WebOb==1.4.1
+
+# MongoDB
+pymongo==2.8.1
```

Comment:
I am not sure I would qualify this fore **core** until it works with the latest major release of pymongo.